src/libs/ErrorUtils.ts

```diff
@@ -61,10 +61,7 @@ type OnyxDataWithErrors = {
 };
 
 function getLatestErrorMessage<TOnyxData extends OnyxDataWithErrors>(onyxData: TOnyxData | null): Localize.MaybePhraseKey {
-    if (!onyxData) {
-        return '';
-    }
-    const errors = onyxData.errors ?? {};
+    const errors = onyxData?.errors ?? {};
```

Comment:
so we don't return an empty string now if there's no `onyxData`?

Reply:
in next line we have
```
 if (Object.keys(errors).length === 0) {
        return '';
    }
```